haystack/backends/elasticsearch_backend.py

```diff
@@ -190,10 +190,20 @@ def update(self, index, iterable, commit=True):
                                extra={"data": {"index": index,
                                                "object": get_identifier(obj)}})
 
-        bulk(self.conn, prepped_docs, index=self.index_name, doc_type='modelresult')
+        try:
+            bulk(self.conn, prepped_docs, index=self.index_name, doc_type='modelresult')
+        except elasticsearch.TransportError as e:
+            if not self.silently_fail:
+                raise
+            self.log.error(u'Error bulk updating Elasticsearch')
```

Comment:
Can we update both of these calls to add the index name, set `exc_info=True`, and see if there's any helpful context to pass to logging? Basically using the call on line 189-192 above as an example?